src/stko/_internal/calculators/xtb_calculators.py

```diff
@@ -87,6 +87,11 @@ class XTBEnergy:
             should be ``True``, however this may raise issues on
             clusters.
 
+        write_sasa_info:
+            If ``True``, the detailed info input will request gbsa=True and
```

Comment:
``gbsa=True`` should be inside of double backticks

Reply:
Done.